Dockerfile

```diff
@@ -23,7 +23,10 @@ COPY api/ api/
 COPY cmd/ cmd/
 COPY controllers/ controllers/
 COPY pkg/ pkg/
+COPY charts/ charts/
+COPY config/ config/
 
+RUN ["make", "helm-repo-index"]
```

Comment:
There is no real advantage to a split make, or is it? You wont get caching from helm charts because the layers above are already changed. WDYT about putting all make targets in the same `RUN`?

Reply:
Sure, I agree there is no real benefit here. Will fix.

Reply:
Fixed.